crates/bevy_feathers/src/focus.rs

```diff
@@ -0,0 +1,105 @@
+//! This module contains the infrastructure needed for displaying focus outlines.
+
+use bevy_app::{Plugin, PostUpdate};
+use bevy_ecs::{
+    change_detection::DetectChanges,
+    component::Component,
+    entity::Entity,
+    hierarchy::Children,
+    query::With,
+    reflect::ReflectComponent,
+    schedule::IntoScheduleConfigs,
+    system::{Commands, Query, Res},
+};
+use bevy_input_focus::{InputFocus, InputFocusVisible};
+use bevy_reflect::{prelude::ReflectDefault, Reflect};
+use bevy_ui::{BorderRadius, GlobalZIndex, Node, Outline, PositionType, UiSystems, Val};
+
+use crate::{theme::UiTheme, tokens};
+
+/// A marker component which indicates that this entity should display a visible focus outline
+/// when either it, or its ancestor, are focused. Insert this into a widget on the entity that
+/// you wisth to display a focus outline.
+#[derive(Component, Default, Clone, Reflect)]
+#[reflect(Component, Clone, Default)]
+pub struct VisibleFocusAnchor;
```

Comment:
The name "Anchor" is usually used to refer to anchor points, maybe something else would be clearer. `VisibleFocusOutline`?

Reply:
I wanted a name that suggests this is *not* an outline, but rather an attachment point for an outline.

Reply:
Maybe `-Mount` or `-Base`? 

I'd prefer an alternative if there's a good one but it's not super important and I'm fine with using `-Anchor` if you aren't happy with anything else. It's just that everywhere else an "anchor" means an attachment at a specific geometric point, instead of fit around this node like it does here.